examples/location_example_parking_garage_opening_hours.json

```diff
@@ -17,15 +17,17 @@
     "uid": "eccb8dd9-4189-433e-b100-cc0945dd17dc",
     "evse_id": "SE*EVC*E000000123",
     "status": "AVAILABLE",
-    "connectors": [{
-      "id": "1",
-      "standard": "IEC_62196_T2",
-      "format": "SOCKET",
-      "power_type": "AC_3_PHASE",
-      "voltage": 230,
-      "amperage": 32,
-      "last_updated": "2017-03-07T02:21:22Z"
-    }],

```

Comment:
Is this only a different layout of the JSON? If we do this, we should do this for all the examples. 
I hope to generate examples in a next version so the layout is the same everywhere. 
So for now I would propose to leave them as is. Or did you change it in all the examples?

Reply:
No it's amperage to max_amperage and voltage to max_voltage.